Языковые курсы.md

```diff
@@ -1,9 +1,16 @@
 ## Интеграционные курсы, языковые курсы:
 
+### Интеграционный курс
 Для понаехавших есть [интеграционные курсы](http://www.bamf.de/RU/Willkommen/DeutschLernen/Integrationskurse/integrationskurse-node.html) по изучению языка
 Школу для курсов можно найти по ссылке http://www.bamf.de/SiteGlobals/Functions/WebGIS/RU/WebGIS_Integrationskursort.html?nn=1441658
+Если вы успешно сдали экхамен после курсов на B1, то вы можете запросить возврат 50% оплаченной стоимости обучения.
```

Comment:
done

2016-03-23 11:34 GMT+01:00 Igor <notifications@github.com>:

> In Языковые курсы.md
> <https://github.com/ewgRa/de_faq/pull/101#discussion_r57137678>:
>
> >  Для понаехавших есть [интеграционные курсы](http://www.bamf.de/RU/Willkommen/DeutschLernen/Integrationskurse/integrationskurse-node.html) по изучению языка
> >  Школу для курсов можно найти по ссылке http://www.bamf.de/SiteGlobals/Functions/WebGIS/RU/WebGIS_Integrationskursort.html?nn=1441658
> > +Если вы успешно сдали экхамен после курсов на B1, то вы можете запросить возврат 50% оплаченной стоимости обучения.
>
> "экхамен" -> "экзамен"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ewgRa/de_faq/pull/101/files/05ebb8af996558eafc6c07d60b19033713dc0cfd#r57137678>
>



-- 

Andrey Tkachenko
Software Engineer

<https://app.adjust.io/aowvd8?campaign=app_DE_iOS_launch&adgroup=email_signature&creative=&deep_link=home24%3A%2F%2F&fallback=http%3A%2F%2Fwww.home24.de%2Fapp%2F>

*www.home24.de* <http://www.home24.de/>

Home24 AG | Greifswalder Straße 212-213 | 10405 Berlin | Deutschland
tel: +49 17 999 31509 | mail: andrey.tkachenko@home24.de

Management Board: Domenico Cipolla (Chairman), Dr. Philipp Kreibohm, Axel
Hefer
Chairman of the Supervisory Board: Lothar Lanz
Seat: Berlin | Registered with the local court: Charlottenburg (Berlin),
HRB 167157 B

Diese Nachricht (inklusive aller Anhänge) ist vertraulich. Sie darf
ausschließlich durch den vorgesehenen Empfänger und Adressaten gelesen,
kopiert oder genutzt werden. Sollten Sie diese Nachricht versehentlich
erhalten haben, bitten wir Sie, den Absender (durch Antwort-E-Mail) hiervon
unverzüglich zu informieren und die Nachricht zu löschen. Jede unerlaubte
Nutzung oder Weitergabe des Inhalts dieser Nachricht, sei es vollständig
oder teilweise, ist unzulässig. Bitte beachten Sie, dass E-Mail-Nachrichten
an den Absender nicht für fristgebundene Mitteilungen geeignet sind.
Fristgebundene Mitteilungen sind daher ausschließlich per Post oder per
Telefax zu übersenden.